lib/shopify_app/engine.rb

```diff
@@ -21,5 +31,16 @@ class Engine < Rails::Engine
         app.config.middleware.insert_after(ShopifyApp::SameSiteCookieMiddleware, ShopifyApp::JWTMiddleware)
       end
     end
+
+    initializer "shopify_app.redact_job_params" do
+      ActiveSupport.on_load(:active_job) do
```

Comment:
Do we need to consider older Rails apps that don't use ActiveJob?

Reply:
ActiveJob was introduced in Rails 4.2 and this gem required 5.2